kafka-commons/src/main/java/uk/gov/homeoffice/digital/sas/kafka/message/KafkaEventMessage.java

```diff
@@ -1,20 +1,24 @@
 package uk.gov.homeoffice.digital.sas.kafka.message;
 
 import jakarta.validation.constraints.NotNull;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 
 @Getter
+@NoArgsConstructor
+@EqualsAndHashCode
 public class KafkaEventMessage<T> {
 
   public static final String SCHEMA_FORMAT = "%s, %s";
 
-  private final String schema;
```

Comment:
Did we need to remove final? Isn't this class used for data reading only?

Reply:
Adding the NoArg constructor caused the error `Variable 'schema' might not have been initialized`
The NoArg constructor is required by jackson library for deserialisation of json into object (e.g. in integration tests)

Reply:
Ah I see. Will resolve.